src/modules/client/components/ClientAlerts.tsx

```diff
@@ -0,0 +1,116 @@
+import { Alert, AlertTitle, Box, Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+import TitleCard from '@/components/elements/TitleCard';
+import theme from '@/config/theme';
+import {
+  clientNameAllParts,
+  parseAndFormatDateTime,
+} from '@/modules/hmis/hmisUtil';
+import {
+  ClientAlertFieldsFragment,
+  ClientFieldsFragment,
+} from '@/types/gqlTypes';
+
+interface ClientAlertCardProps {
+  alert: ClientAlertFieldsFragment;
+  client: ClientFieldsFragment;
+  shouldShowClientName?: boolean;
+}
+
+const ClientAlertCard: React.FC<ClientAlertCardProps> = ({
+  alert,
+  client,
+  shouldShowClientName = false,
+}) => {
+  const priority = alert.severity || 'low';
```

Comment:
Question about language -- I used the word "severity" in the database/backend (following from the figma document "Initial Discovery" section, I think), but the frontend language uses "priority" instead. Should I add another db migration to rename this column (or edit my existing yet-unmerged PR)?

Reply:
Also, should severity/priority become an enum, and if so, where should we enforce that?

Reply:
Lastly, I'm informally setting a default here of "low," but should that be done somewhere more formally on the backend?

Reply:
All good Qs! I think I answered all of that here https://github.com/greenriver/hmis-warehouse/pull/3946#discussion_r1467747598

Whether to rename the DB column is up to you, I think it's fine to leave it

Reply:
There is also a placeholder in the PR description template for linking to the backend PR - that is always helpful for me as a reviewer!

Reply:
Thanks! I linked it.